_sass/_core/typography.scss

```diff
@@ -35,6 +35,7 @@ h4 {
 
 a {
   color: $color-medium;
+  cursor: pointer;
```

Comment:
The whole card is clickable, I noticed a couple of areas where there wasn't a pointer. The top image and some white space between text